storage/app/api/v1/attachments/routes.py

```diff
@@ -29,11 +29,10 @@ async def upload_attachement(
     return controller.upload_attachment_raw(data_file)
```

Comment:
`SpooledTemporaryFile` is all in-memory until a *max_size* is reached, so we might want to attach a small PVC and handle that case in the future